embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/session/SessionHandler.kt

```diff
@@ -111,14 +137,26 @@ internal class SessionHandler(
         completedSpans: List<EmbraceSpanData>? = null
```

Comment:
<i class='graphite__hidden'>[Re: lines 133 to 133]<br/></i>

Why not make this non-null and force the caller to only invoke it if there's an original session? Calling a method that immediately returns makes it seem like the method should't be called at all, and it should be obvious at the call site that this is a no-op<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/74?utm_source=unchanged-line-comment">Graphite</a>.</span>

Reply:
I've moved the null-check higher up.